src/commands.ts

```diff
@@ -620,6 +620,35 @@ export async function getEphemeralPrivKey(index: string) {
     this.resume();
 }
 
+export async function enableDelegatedProver() {
+    this.pause();
+    this.account.setDelegatedProverEnabled(true);
+    this.echo(`Delegated prover enabled`);
```

Comment:
It's need to verify the delegate prover really enabled/disabled

Reply:
setDelegatedProverEnabled throws an error if delegated prover enabling is not possible, so it is not necessary.